### PR TITLE
Prevent soft wrap.

### PR DESCRIPTION
**Pull request by gislenius**

### CODE REVIEW

**Comment by gislenius** [comment]
Beautiful, isn't it? 

http://secxbrl-nowrap.28.io/v1/_queries/public/api/components.jq?token=foobar&aid=0001193125-15-059931&format=html&profile-name=sec

http://secxbrl-nowrap.28.io/v1/_queries/public/api/modelstructure-for-component.jq?token=foobar&aid=0001193125-15-059931&format=html&role=http%3A%2F%2Famericanexpress.com%2Frole%2FStatementConsolidatedStatementsOfIncome&hypercube=xbrl28%3AImpliedTable&profile-name=sec

File: queries/modules/io/28/apps/api.jq [hunk 1]
```diff
@@ -285,7 +285,7 @@ declare function api:csv-to-html(
                 border: 1px solid #ddd;
                 padding: 4px;
                 line-height: 1.42857143;
-                vertical-align: top;
+                vertical-align: top;white-space: nowrap;
```
Comment (dknochen) [inline queries/modules/io/28/apps/api.jq]: could you put it on a seperate line?

**Comment by dknochen** [comment]
simple :)

**Comment by gislenius** [comment]
Simple :)

**Pull request merged**